addons/rose/package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "build:development": "ember build",
     "build": "ember build --environment=production && cp metadata.json dist/metadata.json",
-    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
-    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint": "concurrently \"yarn:lint:*(!fix)\" --names \"lint:\"",
```

Comment:
I run the formatter and shows me 2 file changes, could we run formatter and add the changes to the PR? let me know your thoughts 😉

Reply:
oooo great catch, will do!